os/customizing-sshd.md

```diff
@@ -23,11 +23,11 @@ storage:
           # Use most defaults for sshd configuration.
           UsePrivilegeSeparation sandbox
           Subsystem sftp internal-sftp
+          UseDNS no
 
           PermitRootLogin no
           AllowUsers core
-          PasswordAuthentication no
-          ChallengeResponseAuthentication no
```

Comment:
Ah, no, you're right. This change isn't strictly necessary. Any reason I should prefer the previous config over this though? If not, I'm inclined to keep it as a whitelist.